tests/tests_modifications_with_selinux_disabled.yml

```diff
@@ -0,0 +1,120 @@
+---
+- name: Ensure the default is targeted, enforcing, without local modifications
+  hosts: all
+  gather_facts: true
+  vars:
+    selinux_all_purge: true
+    selinux_logins_purge: true
+
+  tasks:
+    - name: Ensure SELinux tool semanage
```

Comment:
Can you use

```yaml
    - name: Ensure SELinux tool semanage
      include_tasks: set_selinux_variables.yml
```
to ensure semanage is present?

Reply:
`set_selinux_variables.yml` runs 4 time `semanage` command which can take 2 seconds on my vm and information collected by this task is not necessary for the test. As the test is now it doesn't even need `semanage` so it could be dropped completely.

Reply:
> `set_selinux_variables.yml` runs 4 time `semanage` command which can take 2 seconds on my vm

ok - I guess we need a separate task which just installs `semanage`, and another task to populate those test variables.  We can do that in a subsequent commit.  I don't like having multiple copy/paste of the code to install `semanage` - it makes things like `ANSIBLE_GATHERING=explicit`, rpm-ostree support, snapshot creation, new test development, etc. harder.

> and information collected by this task is not necessary for the test. As the test is now it doesn't even need `semanage` so it could be dropped completely.

ack